app/scripts/views/reset_password.js

```diff
@@ -11,14 +11,17 @@ define(function (require, exports, module) {
   const FlowEventsMixin = require('./mixins/flow-events-mixin');
   const FormView = require('./form');
   const PasswordResetMixin = require('./mixins/password-reset-mixin');
-  const preventDefaultThen = require('./base').preventDefaultThen;
   const ServiceMixin = require('./mixins/service-mixin');
   const Session = require('../lib/session');
   const Template = require('templates/reset_password.mustache');
 
   const t = (msg) => msg;
 
-  class ResetPasswordView extends FormView {
+  const ResetPasswordView = FormView.extend({
+    events: {
+      'click .remember-password': '_rememberPassword'
```

Comment:
Do we still need the `preventDefaultThen` here?

Reply:
Not as far as I can tell, unless I'm missing something?

Reply:
I tell a lie, yes we do.